chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -101,12 +101,14 @@
 
 <!ENTITY zotero.preferences.quickCopy.caption				"Quick Copy">
 <!ENTITY zotero.preferences.quickCopy.defaultOutputFormat	"Default Output Format:">
+<!ENTITY zotero.preferences.quickCopy.locale				"Bibliography Locale:">
```

Comment:
I can hardly find an example of a software/OS which uses the term "locale" in its UI…

"Language and Text" (Mac OS, Ubuntu), "Regional and Language Options" (Windows), "Language and Region" (iOS), "Language & Keyboard" (Android, which had used "Locale & Text"… <a href="http://kb.intuit.com/library/INTUIT/SBG/PSD/Jessika/GoPay%20Android%20Keyboard%20Settings%200010.jpg">before v.2.0</a>), iTunes uses "Languages & Text" too (<a href="https://support.apple.com/en-us/HT201690">second screenshot and table</a>).

In these contexts, locale could have been used because it's exactly that : a language and a region/location/culture which have an impact on some parameters (numbers, date and time, currency…). But it's not.

That's because "Locale" is somewhat <a href="http://en.wikipedia.org/wiki/Locale">techy</a> jargon-laden, even if we're just used to it…

Also, it is never used, for instance, in the wikipedia articles about British and American English (but you'll find "dialect", "variety" or "form" – and "language" obviously) [<a href="http://en.wikipedia.org/wiki/Comparison_of_American_and_British_English">1</a>, <a href="http://en.wikipedia.org/wiki/List_of_dialects_of_the_English_language">2</a>, <a href="http://en.wikipedia.org/wiki/General_American">3</a>, <a href="http://en.wikipedia.org/wiki/Standard_English">4</a>, <a href="http://en.wikipedia.org/wiki/English_language#Dialects.2C_accents.2C_and_varieties">5</a>, <a href="http://en.wikipedia.org/wiki/American_English">6</a>…].